splunk_eventgen/lib/eventgenconfig.py

```diff
@@ -520,15 +521,22 @@ def parse(self):
             if os.path.exists(s.sampleDir):
                 sampleFiles = os.listdir(s.sampleDir)
                 for sample in sampleFiles:
-                    results = re.match(s.name, sample)
+                    sample_name = s.name
+                    # If we expect a .csv, append it to the file name - regex matching must include the extension
+                    if s.sampletype == "csv" and not s.name.endswith(".csv"):
+                        sample_name = s.name + ".csv"
```

Comment:
Maybe escape `.` will be more accurate to find the CSV sample file. 
For example, we do not want to match `abcdcsv`, however, we want to extract the `.csv` literally.

```
sample_name = s.name + "\.csv"
```

@jmeixensperger  what's your thought?

Reply:
I agree, I'll test it out and make sure it works.